clang-tools-extra/test/clang-tidy/checkers/bugprone/non-zero-enum-to-bool-conversion.cpp

```diff
@@ -126,4 +126,10 @@ void testCustomOperator(CustomOperatorEnum e) {
     if (!(e & E1)) {}
 }
 
+enum EmptyEnum {};
+
+bool testCustomOperator(EmptyEnum value) {
```

Comment:
why we should ignore this cases?

Reply:
Because this is enum without enumerators = can hold any value = usually used to work as strong type